packages/backend/src/tests/propertyBased/arbitraries/cardinality.ts

```diff
@@ -0,0 +1,109 @@
+import { fc } from '@fast-check/vitest';
+import {
+    AnySchema,
+    getSelectableColumnsFromGenericDbSchema,
+    getWhereableColumnsFromGenericDbSchema,
+    getTableNamesFromGenericDbSchema,
+} from '@synthql/queries';
+
+type Cardinality = 'one' | 'maybe' | 'many';
+
+export function generateQueryArbitrary(
+    schema: AnySchema,
+    allValuesMap: Map<string, Array<any>>,
+    cardinality: Cardinality,
+) {
+    return fc
+        .constantFrom(...getTableNamesFromGenericDbSchema(schema))
+        .chain((table) =>
```

Comment:
Let's split this function up into more granular, easier to manage arbitraries. Let's also use a naming convention to easily identify them:

For example:

1. `function arbitraryTablename(schema: AnySchema)`
2. `arbitrarySelect(schema, table)`
3. `arbitraryWhere(...)`
4. `arbitraryLimit(...)`
5. `arbitraryCardinality(...)`

Reply:
This also means using the `arbitrary` prefix to name the top-level arbitraries like `arbitraryQuery` and `arbitraryEmptyQuery`